maflib/schemas/gdc-2.0.0-aliquot.json

```diff
@@ -0,0 +1,118 @@
+{
+	"version" : "gdc-1.0.0",
```

Comment:
Should these versions stay `1.0.0` or also be bumped?

Reply:
that is for the "base" MAF spec, the first 30-something columns which have not changed since inception, the other version tag is for all the other columns which is what was bumped to the 2.0.0